compiler_opt/rl/inlining/config.py

```diff
@@ -99,3 +103,54 @@ def observation_processing_layer(obs_spec):
 
 def get_nonnormalized_features():
   return ['reward', 'inlining_default', 'inlining_decision']
+
+
+class InliningForSizeTask(env.MLGOTask):
```

Comment:
can you move this to its own file, like an inlining_task.py, config.py has just the ml-facing things.